tests/providers/jira/operators/test_jira.py

```diff
@@ -39,8 +38,11 @@
 }
 
 
-class TestJiraOperator(unittest.TestCase):
-    def setUp(self):
+class TestJiraOperator:
+    def setup_class(self):
+        clear_db_runs()
```

Comment:
Are these changes related/needed cos of the change in this PR or something else?